src/styles/blocks/card.scss

```diff
@@ -0,0 +1,135 @@
+.catalog {
+  padding: 50px 40px;
+  gap: 46px 48px;
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
+  grid-auto-flow: row;
+  align-items: center;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-size);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-size);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-size);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__image {
+    margin: 0 auto;
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $text-main-color;
+  }
+
+  &__code{
+    font-weight: 400;
```

Comment:
It is default value, so we don`t need to specify it additionally (check it everywhere)